app/navigation/Navigator.scala

```diff
@@ -242,4 +242,15 @@ class Navigator @Inject()() {
     case Some(false) => NoFurtherActionController.onPageLoad()
     case _ => SessionExpiredController.onPageLoad()
   }
+
+  private def checkYourAnswers(userAnswers: UserAnswers): Call = {
+    userAnswers.get(SummarySubscriptionsPage)(PSubsByYear.formats).map(_.filter(_._2.nonEmpty).keys.toSeq) match {
+      case Some(years) =>
+        years match {
+          case years if !years.contains(getTaxYear(CurrentYear)) => HowYouWillGetYourExpensesController.onPageLoad()
+          case _ => YourEmployerController.onPageLoad(NormalMode)
```

Comment:
Would this not read better to have:
case years if years.contains(getTaxYear(currentYear)) => YourEmployerController...
case _ => HowYou.......